src/content/popup/metadata.test.ts

```diff
@@ -23,7 +23,9 @@ vi.mock('../../common/i18n', async () => {
 
 import { ShogiMeta } from '../shogi';
 
-import { renderMetadata } from './metadata';
+import { h, render } from 'preact';
+import { ShogiInfo } from './Metadata/ShogiInfo';
+import { html } from '../../utils/builder';
```

Comment:
I think most of these imports could go to the type of the file (and be sorted in the regular order). `ShogiMeta` needs to come after the mock (I believe) and maybe `ShogiInfo` does too? But the others, I think, shouldn't be affected.

Also, it's probably not worth importing `html` from the HTML builder utility since ultimately we'll probably drop that once we've converted everything to Preact (and we only use it here to create a single `<div>` as far as I can tell).

Reply:
> I think most of these imports could go to the top of the file (and be sorted in the regular order). `ShogiMeta` needs to come after the mock (I believe) and maybe `ShogiInfo` does too? But the others, I think, shouldn't be affected.

Oh, my editor put them there automatically.

Interestingly, the tests pass even without `// sort-imports-ignore` with every import on top of the file, before the mock. Can I just drop it?

Reply:
Oh nice! I _think_ there was an update to vitest at some point to make mocking ESM modules work better so maybe it used to be necessary? If it's not necessary, yes, let's drop the `sort-imports-ignore` and use the prettier-defined sorting.

Reply:
Yeah, it probably used to be necessary. I dropped it now.